src/test/resources/net/sf/jabref/importer/fileformat/SilverPlatterImporterTest2.bib

```diff
@@ -0,0 +1,9 @@
+@Article{,
+publisher = {New York publishment}, 
+year = {3 Jun},
+keywords = {many-much-more-i don't know},
+school = {Gymnasium Unterrieden},
+editor = {Supek-S and Aine-CJ},
+title = {imulation studies of multiple dipole neuromagnetic "\},
```

Comment:
The title will only be saved till an " in comes. The code herefore is 
`                         int inPos = title.indexOf("\" in ");
                         if (inPos > 1) {
                             h.put("title", title.substring(1, inPos));
                         }`
in the SilverplatterImporter